bot/cogs/sync/cog.py

```diff
@@ -26,58 +26,72 @@ class Sync:
         syncers.sync_users
     )
 
-    def __init__(self, bot):
+    def __init__(self, bot: Bot) -> None:
         self.bot = bot
 
-    async def on_ready(self):
+    async def on_ready(self) -> None:
+        """Syncs the roles/users of the guild with the database."""
         guild = self.bot.get_guild(self.SYNC_SERVER_ID)
         if guild is not None:
             for syncer in self.ON_READY_SYNCERS:
                 syncer_name = syncer.__name__[5:]  # drop off `sync_`
                 log.info("Starting `%s` syncer.", syncer_name)
-                total_created, total_updated = await syncer(self.bot, guild)
-                log.info(
-                    "`%s` syncer finished, created `%d`, updated `%d`.",
-                    syncer_name, total_created, total_updated
-                )
-
-    async def on_guild_role_create(self, role: Role):
+                total_created, total_updated, total_deleted = await syncer(self.bot, guild)
+                if total_deleted is None:
+                    log.info(
+                        "`%s` syncer finished, created `%d`, updated `%d`.",
```

Comment:
I thought these were changed to f-strings in a previous commit but maybe that was something else?

I'm generally not a fan of this interpolation style, if we don't have a good reason for using it I'd prefer we switch it out with something more readable for everything in this file.

There _are_ legitimate reasons for using this (for example if you're interpolating something very heavy and you're doing it at loglevel info and we're running loglevel error in production, that heavy thing will never be evaluated), but I don't think they apply to any of these.

Reply:
I must have fucked somewhere or did not actually change the code after the agreement we had on it.

Reply:
As far as I can tell, I just completely forgot to change the code after our discussion in dev-core.